package.json

```diff
@@ -4,9 +4,10 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "npm run format && next build",
```

Comment:
I think this is too opinionated for the `build` command. Can you move it as part of `format`?